### PR TITLE
[SSL] Update for Automatic ssl/tls

### PR DESCRIPTION
**Pull request by kodster28**
### Summary

Updates based on https://blog.cloudflare.com/introducing-automatic-ssl-tls-securing-and-simplifying-origin-connectivity

### CODE REVIEW

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 1]
```diff
@@ -7,28 +7,75 @@ head: []
...
+  --header 'Authorization: Bearer <CF_API_TOKEN>' \
+  --header 'Content-Type: application/json' \
+  --data '{"value":"custom"}'
+```
+
+#### Opt out multiple zones
+
+If you wanted to opt out multiple zones:
+
+1. Create an API token with the following permissions:
```
Comment (kodster28) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx:58]: Grabbed and modified from existing place in our docs at https://developers.cloudflare.com/fundamentals/setup/manage-domains/add-multiple-sites-automation/#add-domains

**Review by crwaters16 — APPROVED** [review]

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 2]
```diff
@@ -7,28 +7,75 @@ head: []
...
 
+### Automatic SSL/TLS (default)
 
-:::
+Automatic SSL/TLS uses the [SSL/TLS Recommender](/ssl/origin-configuration/ssl-tls-recommender/) to identify and apply the most secure encryption mode for your website. It continuously assesses your traffic to determine if a more secure option is needed based on your origin certification or capabilities, and updates your settings automatically to ensure the highest level of security for your domain.
 
-## Available encryption modes
+#### Opt out single zone
+
+If you want to opt a zone out via the API, you can make this API call on or before the grace period expiration date (September 16, 2024).
```
Comment (yj7o5) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx]:
```suggestion
If you want to opt a zone out via the API, you can make this API call on or before the grace period expiration date (September 16th, 2024).
```

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 3]
```diff
@@ -7,28 +7,75 @@ head: []
...
+```
+
+#### Opt out multiple zones
+
+If you wanted to opt out multiple zones:
+
+1. Create an API token with the following permissions:
+   - `Zone - Zone Settings - Read`
+   - `Zone - Zone Settings - Edit`
+2. Create a list of zone IDs you want to each, each on a separate line (newline separate), stored in a file such as `zones.txt`.
```
Comment (yj7o5) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx]:
```suggestion
2. Create a list of zone IDs you want to opt-out with each zone ID on a separate line (newline separate), stored in a file such as `zones.txt`.
```

**Comment by yj7o5** [comment]
@kodster28 can we add a deprecation notice to https://automatic-ssl-mode.cloudflare-docs-7ou.pages.dev/ssl/origin-configuration/ssl-tls-recommender/#enable-ssltls-recommendations as Automatic SSL/TLS since once after all the zones have been migrated we won't have the SSL/TLS Recommender dash UI visible anymore nor the API routes for SSL/TLS recommender. It will just be Automatic SSL/TLS.

**Comment by kodster28** [comment]
> @kodster28 can we add a deprecation notice to https://automatic-ssl-mode.cloudflare-docs-7ou.pages.dev/ssl/origin-configuration/ssl-tls-recommender/#enable-ssltls-recommendations as Automatic SSL/TLS since once after all the zones have been migrated we won't have the SSL/TLS Recommender dash UI visible anymore nor the API routes for SSL/TLS recommender. It will just be Automatic SSL/TLS.

This is done

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 4]
```diff
@@ -17,15 +17,65 @@ If possible, Cloudflare strongly recommends using [**Full**](/ssl/origin-configu
...
 :::
 
-## Available encryption modes
+### Automatic SSL/TLS (default)
+
+Automatic SSL/TLS uses the [SSL/TLS Recommender](/ssl/origin-configuration/ssl-tls-recommender/) to identify and apply the most secure encryption mode for your website. It continuously assesses your traffic to determine if a more secure option is needed based on your origin certification or capabilities, and updates your settings automatically to ensure the highest level of security for your domain.
+
+#### Opt out single zone
+
+If you want to opt a zone out via the API, you can make this API call on or before the grace period expiration date (September 16th, 2024).
```
Comment (yj7o5) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx]: The sept 16th isn't right here. We actually have it broken down by plans can we instead have a table that says something like this:

|plan|ssl recommender users|grace period expiration date|
|---|---|---|
|free/paygo|yes|sept 9th|
|ent|yes|oct 7th|
|free/paygo/ents|no|oct 21st|

what this shows is that free/pro/biz customers that had recommender enabled will have grace period of sept 9th, while ent customers that had recommender enabled will have grace period of oct 7th while all the rest of the zones that never enrolled into ssl recommender will have grace period of oct 21st. @kodster28

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 5]
```diff
@@ -17,15 +17,65 @@ If possible, Cloudflare strongly recommends using [**Full**](/ssl/origin-configu
...
-If you are not sure which encryption mode to use, enable the [SSL/TLS Recommender](/ssl/origin-configuration/ssl-tls-recommender/).
+:::note
+We are gradually rolling out the new [Automated SSL/TLS feature](#automatic-ssltls-default).
 
+If your zone has not been migrated yet, you will only have [Custom SSL/TLS](#custom-ssltls) options in your dashboard.
 :::
 
-## Available encryption modes
+### Automatic SSL/TLS (default)
+
```
Comment (yj7o5) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx:31]: can we also explain the difference between "automatic" and "custom" mode? we explain the difference here https://blog.cloudflare.com/introducing-automatic-ssl-tls-securing-and-simplifying-origin-connectivity#:~:text=Automatic%20SSL/TLS%20uses%20the%20SSL/TLS%20Recommender so if could briefly describe here as well then it'd be useful.

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 6]
```diff
@@ -17,15 +17,65 @@ If possible, Cloudflare strongly recommends using [**Full**](/ssl/origin-configu
...
+```
+
+#### Opt out multiple zones
+
+If you wanted to opt out multiple zones:
+
+1. Create an API token with the following permissions:
+   - `Zone - Zone Settings - Read`
+   - `Zone - Zone Settings - Edit`
+2. Create a list of zone IDs you want to opt-out with each zone ID on a separate line (newline separate), stored in a file such as `zones.txt`.
```
Comment (yj7o5) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx]: can we add the possibly using of this route https://developers.cloudflare.com/api/operations/zones-get to get all the zone ids to build up the list? say a customer wanted to opt-out of all the zones under an account to utilize the following API @kodster28

**Review by yj7o5 — COMMENTED** [review]
Left a few more remarks

**Comment by kodster28** [comment]
> Left a few more remarks

Okay, should be addressed by the latest pushes

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 7]
```diff
@@ -17,15 +17,84 @@ If possible, Cloudflare strongly recommends using [**Full**](/ssl/origin-configu
...
-## Available encryption modes
+### Automatic SSL/TLS (default)
+
+Automatic SSL/TLS uses the [SSL/TLS Recommender](/ssl/origin-configuration/ssl-tls-recommender/) to identify and apply the most secure encryption mode for your website. It continuously assesses your traffic to determine if a more secure option is needed based on your origin certification or capabilities, and updates your settings automatically to ensure the highest level of security for your domain.
+
+Cloudflare is enabling Automatic SSL/TLS on the following dates:
+
+| Plan           | Using SSL/TLS recommender? | Grace period ends |
+| -------------- | -------------------------- | ----------------- |
+| Non-Enterprise | Yes                        | September 9, 2024 |
```
Comment (yj7o5) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx]:
```suggestion
| Non-Enterprise | Yes                        | September 9th, 2024 |
```

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 8]
```diff
@@ -17,15 +17,84 @@ If possible, Cloudflare strongly recommends using [**Full**](/ssl/origin-configu
...
+| Enterprise     | Yes                        | October 7, 2024   |
```
Comment (yj7o5) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx]:
```suggestion
| Enterprise     | Yes                        | October 7th, 2024   |
```

File: src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx [hunk 9]
```diff
@@ -17,15 +17,84 @@ If possible, Cloudflare strongly recommends using [**Full**](/ssl/origin-configu
...
+| All plans      | No                         | October 21, 2024  |
```
Comment (yj7o5) [inline src/content/docs/ssl/origin-configuration/ssl-modes/index.mdx]:
```suggestion
| All plans      | No                         | October 21st, 2024  |
```

**Review by yj7o5 — COMMENTED** [review]
All looks good, just left 1 minor edit and then we should be good to go.

**Comment by yj7o5** [comment]
all looks good, approved!

**Pull request merged**